Detectors/GlobalTracking/include/GlobalTracking/MatchTOF.h

```diff
@@ -194,6 +196,11 @@ class MatchTOF
   void setTS(unsigned long creationTime) { mTimestamp = creationTime; }
   unsigned long getTS() const { return mTimestamp; }
 
+  static void groupingMatch(std::vector<o2::dataformats::MatchInfoTOFReco> origin, std::vector<std::vector<o2::dataformats::MatchInfoTOFReco>>& grouped, std::vector<std::vector<int>>& firstEls, std::vector<std::vector<int>>& secondEls);
```

Comment:
passing `origin` by copy, see more comments in the cxx file.

Reply:
Sure. Moving to const reference